src/rust/plugin-registry/src/static_files/hax_docker_generator.nomad

```diff
@@ -138,6 +143,8 @@ job "grapl-plugin" {
   }
 
   group "plugin" {
+    consul { namespace = local.namespace }
```

Comment:
This may not work as expected since per the docs it appears to be enterprise only: https://www.nomadproject.io/docs/job-specification/group#namespace

Reply:
Actually @wimax-grapl what you probably want is to run the entire job in a Nomad namespace, which is supported per https://learn.hashicorp.com/tutorials/nomad/namespaces#run-a-job-in-a-namespace

Reply:
Surprise! Nomad namespaces are not the same as Consul namespaces. (However, the jobs already have their own Nomad namespace with the same name)